app/helpers/application_helper.rb

```diff
@@ -1,5 +1,25 @@
 module ApplicationHelper
-  def ficon(name, size = nil)
-    content_tag(:i, nil, class: "fi-#{name} #{size} icon")
+  def ficon(name, size = nil, *classes)
+    add = classes.map(&:to_s).join(' ')
+    content_tag(:i, nil, class: "fi-#{name} #{size} icon #{add}")
+  end
+
+  def nav_link(icon = nil, name = nil, options = nil, html_options = nil, &block)
+    html_options, options, name = options, name, block if block_given?
+    options ||= {}
+    html_options = convert_options_to_data_attributes(options, html_options)
+    url = url_for(options)
+    html_options['href'] ||= url
+    concat (
+      content_tag(:div, class: 'row') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.